operations/services.md

```diff
@@ -1157,7 +1155,7 @@ Creates a new order with the specified products and items. Only positive charges
 | `Client` | string | required | Name and version of the client application. |
 | `CustomerId` | string | required | Identifier of the [Customer](customers.md#customer) to be charged. |
 | `ServiceId` | string | required | Identifier of the [Service](services.md#service) to be ordered. |
-| `ConsumptionUtc` | string | optional | Date and time of the order consumption in UTC timezone in ISO 8601 format. If not specified, current date and time is used. |
+| `ConsumptionUtc` | string | optional | Date and time of the order consumption in UTC timezone in ISO 8601 format. If not specified, current date and time is used. Please note, as order consumption is one-time event, the optional parameters `StartUtc` and `EndUtc` in [Product order parameters](services.md#product-order-parameters) should not be used. |
```

Comment:
I think in commander, you are able to specify start. Only end shouldn't be used. So I propose to support just start. Consumption of order is not directly related to product order start/end so I'd exclude it from here.